lib/player.js

```diff
@@ -435,6 +435,26 @@ goog.requireType('shaka.routing.Payload');
  */
 
 
+/**
+ * @event shaka.Player.KeyStatusChanged
+ * @description Fired when the key status changed.
+ * @property {string} type
+ *   'keystatuschanged'
+ * @exportDoc
+ */
+
+
+/**
+ * @event shaka.Player.UpdateState
+ * @description Fired when player state is updated.
+ * @property {string} type
+ *   'updatestate'
```

Comment:
It is more conventional to call a change event something like "statechanged", as you did for "keystatuschanged".

Reply:
Done!